src/tests/ftest/pool/list_pools.py

```diff
@@ -35,47 +36,71 @@ def run_case(self, rank_lists, svcn=None):
         # Iterate rank lists to create pools. Store the created pool information
         # as a dictionary of pool UUID keys with a service replica list value.
         self.pool = []
-        expected_uuids = {}
+        expected_admin_uuids = {}
         for rank_list in rank_lists:
             self.pool.append(self.get_pool(target_list=rank_list, svcn=svcn))
-            expected_uuids[self.pool[-1].uuid.lower()] = self.pool[-1].svc_ranks
-
-        # Verify the 'dmg pool info' command lists the correct created pool
-        # information.  The DmgCommand.pool_info() method returns the command
-        # output as a dictionary of pool UUID keys with service replica list
-        # values.
-        detected_uuids = {}
+            expected_admin_uuids[self.pool[-1].uuid.lower()] = self.pool[-1].svc_ranks
+
+        # Remove the default ACLs that allow the creator to access the pool.
+        # These ACLs don't apply to dmg, but do apply to users.
+        offlimits = self.pool[0]
+        self.get_dmg_command().pool_delete_acl(offlimits.uuid, 'OWNER@')
+        self.get_dmg_command().pool_delete_acl(offlimits.uuid, 'GROUP@')
```

Comment:
We do have class methods for this
```suggestion
        offlimits.delete_acl('OWNER@')
        offlimits.delete_acl('GROUP@')
```

Reply:
Lost the merge race and had to deal with a conflict, so I went ahead and made this change. Definitely cleaner this way.